charts/datasaur/templates/jobs/migrator-job.yaml

```diff
@@ -10,7 +10,19 @@ spec:
     metadata:
       labels:
         app: {{ .Release.Name }}-migrator
+      annotations:
+        "helm.sh/hook": post-install
+        "helm.sh/hook-weight": "1"
     spec:
+      initContainers:
+        - name: wait-for-mariadb-before-starup
+          image: busybox
+          command:  ["sh", "-c", "until nc -z {{ .Release.Name }}-mariadb-service 3306 > /dev/null; do echo Waiting for mariadb.; sleep 2; done;"]
```

Comment:
does this work for RDS or managed service?

Reply:
ah iya juga ya, harus diganti brati jadi DATABASE_HOST

Reply:
decided to remove this and fallback to retries, best practice nya ternyata begitu.